app/models/sample/condition.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+# model to represent sample search condition
+class Sample::Condition # rubocop:disable Style/ClassAndModuleChildren
```

Comment:
This class should be renamed to `Sample::SearchCondition` for clarity since that's what this file represents